### PR TITLE
Added a note on how to contribute in the README

### PR DESCRIPTION
**Pull request by phillip-kruger**
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

As discussed in the community meeting, more details on how to contribute.

### CODE REVIEW

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

**Comment by phillip-kruger** [comment]
Moved the How to Contribute to it's own file as discussed here:
https://groups.google.com/forum/#!topic/microprofile/-o0Mqv4I2IY

**Comment by Emily-Jiang** [comment]
Pls see the [discussion](https://groups.google.com/forum/#!topic/microprofile/-o0Mqv4I2IY) I suggest to move the content under microprofile.io, which is more visible and easy to maintain.

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+// limitations under the License.
+//
+
+== How to contribute
+
+Do you want to contribute to this project? Here is what you can do:
+
+* Fork the repository, make changes, then do a pull request.
+** Remember to https://help.github.com/articles/signing-commits/[sign your commits]
+** Make sure you have signed the https://www.eclipse.org/legal/ECA.php[Eclipse Contributor Agreement]
```
Comment (struberg) [inline CONTRIBUTING.adoc]: ECA is only required for biggish changes which create originary IP. And even then any contribution would legally still be covered by our ALv2.

**Comment by Emily-Jiang** [comment]
The info for meeting time is wrong. I am confused why this PR was merged despite of requesting not to merge as yet @struberg ?

**Comment by struberg** [comment]
We discussed it in the last mp-config hangout and decided to apply it. The chnange is basically good. If there are smallish glitches then we gonna fix them as normal change. We have PRs open for months without applying them nor giving proper feedback. This is not good.

**Pull request merged**